etc/helm/pachyderm/templates/enterprise-server/deployment.yaml

```diff
@@ -140,6 +140,24 @@ spec:
             configMapKeyRef:
               name: "pachyderm-identity-config"
               key: "oidc-clients"
+        {{- if or .Values.oidc.additionalOIDCClientsSecretName .Values.oidc.additionalOIDCClients }}
+        - name: IDP_CLIENTS_EXTRAS
```

Comment:
perhaps we should stick with `additional_clients` here for consistency?

Reply:
good call 👍